### PR TITLE
Miscellaneous small fixes

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt [hunk 1]
```diff
@@ -122,9 +122,9 @@ data class ResolvedLicenseInfo(
         licenseChoices: List<SpdxLicenseChoice>,
         licenseView: LicenseView = LicenseView.ALL
     ): ResolvedLicenseInfo {
-        val licenses = effectiveLicense(licenseView, licenseChoices)?.decompose().orEmpty()
+        val effectiveLicenses = effectiveLicense(licenseView, licenseChoices)?.decompose().orEmpty()
```
Comment (mnonnenmacher) [inline model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt:125]: Unrelated renaming.

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt (hunk 1 above)
Comment (sschuberth) [reply model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt:125]: It's not unrelated: In order to be able to simplify `this.licenses`, the local `licenses` has to be renamed.

**Review by mnonnenmacher — APPROVED** [review]
